rasa/core/policies/ted_policy.py

```diff
@@ -255,7 +258,7 @@ class TEDPolicy(Policy):
         TENSORBOARD_LOG_DIR: None,
         # Define when training metrics for tensorboard should be logged.
         # Either after every epoch or for every training step.
-        # Valid values: 'epoch' and 'minibatch'
+        # Valid values: 'epoch' and 'batch'
```

Comment:
Also, should we add/modify a test to use `batch` instead of `minibatch`? However, not sure what would the test do since it's hard to actually check the logs.

Reply:
We automatically update `minibatch` to `batch` and print a warning. But will add it to the changelog and migration guide. Will take a look at our tests.